SimPEG/Examples/Inversion_IRLS.py

```diff
@@ -0,0 +1,132 @@
+from SimPEG import *
+
+
+def run(N=200, plotIt=True):
+    """
+        Inversion: Linear Problem
```

Comment:
I would suggest a title along the lines of `Linear Inversion using Iteratively Reweighted Least Squares`